### ISSUE TITLE
non existent ivoatex/startup.sh

### ISSUE THREAD

**Issue by molinaro-m** [ivoa-std/ivoatexDoc#30]
Sec. 2.2.2 of ivoatexDoc mentions an `ivoatex/startup.sh` script that ivoatex submodule does not (any longer?) include.

Maybe the `ivoatex/make-template.sh` supersedes it?

**Comment by msdemlei** [comment]
On Thu, Jun 01, 2023 at 05:47:11AM -0700, Marco Molinaro wrote:
> Sec. 2.2.2 of ivoatexDoc mentions an `ivoatex/startup.sh` script that ivoatex submodule does not (any longer?) include.
>
> Maybe the `ivoatex/make-template.sh` supersedes it?

Yes; actually, it was only called "startup.sh" for the duration of a
pull request, but as we settled for the new name, I only fixed the
example, not the name mentioned in the running text.

This is fixed in commit 586de639e7b8ba0ea6af7a55e592a196db5ab151.

Which makes we wonder: Perhaps we should release a new ivoatexDoc?
There's not overly many changes over the 1.3 that's currently online,
but mentioning the new-release target, a few bug fixes, and a few
extra examples on generated content might already be worth the effort
of pushing a new version?
